src/locales/cy/translation.json

```diff
@@ -1019,28 +1019,19 @@
       "content": {
         "paragraph": "Bydd hyn yn ein helpu i ddod o hyd i’r ffordd orau i chi brofi eich hunaniaeth.",
         "subHeading": "Trwydded yrru cerdyn gyda llun y DU",
-        "paragraph1": "Gallwch ddefnyddio trwydded yrru lawn neu dros dro y DU os yw’n:",
-        "drivingLicenceRequirements": [
-          "gerdyn-llun plastig, ddim yn drwydded papur",
-          "heb ddod i ben"
-        ],
-        "subHeading2": "Pasbort y DU",
-        "paragraph3": "Gallwch ddefnyddio unrhyw pasbort y DU. Os yw’ch pasbort y DU wedi dod i ben, gallwch barhau i’w ddefnyddio i brofi eich hunaniaeth hyd at 18 mis ar ôl ei ddyddiad dod i ben.",
-        "subHeading3": "Pasbort o’r tu allan i’r DU gyda sglodion biometrig",
-        "paragraph4": "Gallwch ddefnyddio pasbort o’r tu allan i’r DU os:",
-        "nonUkPassportRequirements": [
-          "mae ganddo sglodyn biometrig",
-          "nid yw wedi dod i ben"
+        "paragraph1": "Gallwch ddefnyddio un o’r mathau canlynol o ID gyda llun:",
+        "documentsList": [
```

Comment:
We have a reusable block for this "typesOfPhotoId.appDocumentList", these translations should be moved there

Reply:
If we could use the component here I'd agree, but I think if we want to re-use the translations here we should pull them out into a component and use that rather than use the same translation text in two separate places.